### DISCUSSION TITLE
Open specific dashboard after pimcore admin login

### DISCUSSION THREAD

**Discussion by danielczestki** [pimcore/pimcore#9416] [Q&A / Support]
Hello.

I want to setup redirect to specific dashboard after login to pimcore.
Is it possible ? How ? I tried to use redirects but without success.

**Comment by dvesh3** [comment]
@danielczestki Did you try setting dashboard in perspective config? https://github.com/pimcore/demo/blob/cadc7dd61a776058137b726efb44f0a0cdbac25a/config/pimcore/perspectives.php#L29
